src/os/ObjectStore.h

```diff
@@ -872,7 +872,6 @@ class ObjectStore {
     Op* _get_next_op() {
       if (op_ptr.length() == 0 || op_ptr.offset() >= op_ptr.length()) {
         op_ptr = bufferptr(sizeof(Op) * OPS_PER_PTR);
-	op_ptr.zero();
```

Comment:
I think in the latter case, it would create new op_ptr even if another op would fit into existing op_ptr (i. e. op_ptr.offset + sizeof(Op) == op_ptr.length()). Besides, op_ptr.offset() is a multiple of sizeof(Op), so in worst case it should be equal to op_ptr.length(), so the condition will work properly.